pkg/util/logs/logs.go

```diff
@@ -44,12 +44,22 @@ func (writer GlogWriter) Write(data []byte) (n int, err error) {
 
 // InitLogs initializes logs the way we want for kubernetes.
 func InitLogs() {
+	logFlushFreq := pflag.Duration("log-flush-frequency", DefaultLogFlushFreq, "Maximum number of seconds between log flushes")
```

Comment:
> Whats the motivation for changing this file?

@mikedanese it's to avoid having `--log-flush-frequency` being added. I've opened a PR (https://github.com/kubernetes/kubernetes/pull/32208) to fix it, take a look there is another issue about it already... Also, @luxas showed me the function to hide flags, which we can use to, but seemed like it's still a good idea to attempt fixing the cause.

> Since you've moved this out into a separate PR, let's remove it from here

@luxas I suppose I can drop it from here and have a branch for testing where I'll cherry-pick all the changes that make it build and work.